drivers/sensor/bh1750/Kconfig

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2022 Lucas Heitzmann Gabrielli
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig BH1750
+	bool "Rohm BH1750 16 bit ambient light sensor"
+	default y
+	depends on DT_HAS_ROHM_BH1750_ENABLED
+	depends on I2C
+	help
+	  Enable driver for the Rohm BH1750 pressure/altitude sensor
+
+if BH1750
+
+choice
+	prompt "BH1750 resolution mode"
```

Comment:
This option should be converted to a devicetree property to allow configuring it on a per-instance basis